jms/1.7/modules/ROOT/pages/jms-studio-configuration.adoc

```diff
@@ -0,0 +1,478 @@
+= Use Studio to Configure JMS Connector 1.7 - Mule 4
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-connection,Configure the connection to a broker>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:jms-connector-reference.adoc[JMS Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Anypoint Connector for JMS (JMS Connector) to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `jms` in the search field.
+. Click *JMS Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with JMS Connector:
+
+* *JMS > On New Message* +
+Initiates a flow by listening for incoming messages
+* *HTTP > Listener* +
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler* +
+Initiates a flow when a time-based condition is met
```

Comment:
Because these are short...:
```suggestion
* *JMS > On New Message*: Initiates a flow by listening for incoming messages
* *HTTP > Listener*: Initiates a flow each time it receives a request on the configured host and port
* *Scheduler*: Initiates a flow when a time-based condition is met
```
Otherwise, list items require a blank line and new para for descriptions, or use of a Definition List. GLOBAL.